bitcoin/examples/create-p2wpkh-address.rs

```diff
@@ -0,0 +1,23 @@
+extern crate bitcoin;
+
+use bitcoin::{KnownHrp, Address, CompressedPublicKey, PrivateKey, Network};
+use bitcoin::secp256k1::{rand, Secp256k1};
+
+fn main() {
+    // Create a new Secp256k1 elliptic curve.
+    let s = Secp256k1::new();
+
+    // Generate a Secp256k1 secret key.
+    let (secret_key, _) = s.generate_keypair(&mut rand::thread_rng());
```

Comment:
In 77d8b4793cab782dc2e47c867b2a3e29f08eb884:

Here you generate a public key and throw it away...